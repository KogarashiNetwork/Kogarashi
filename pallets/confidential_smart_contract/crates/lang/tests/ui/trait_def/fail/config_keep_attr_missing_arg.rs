use ink_lang as ink;

#[ink::trait_definition(keep_attr)]
pub trait TraitDefinition {
    #[ink(message)]
    fn message(&self);
}

fn main() {}
