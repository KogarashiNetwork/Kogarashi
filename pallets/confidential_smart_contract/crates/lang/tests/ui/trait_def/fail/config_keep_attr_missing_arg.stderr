error: ink! config options require an argument separated by '='
 --> tests/ui/trait_def/fail/config_keep_attr_missing_arg.rs:3:25
  |
3 | #[ink::trait_definition(keep_attr)]
  |                         ^^^^^^^^^
